templates/company/index.html

```diff
@@ -0,0 +1,173 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{% block title %}Company{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-light-grey">
+  <div class="u-fixed-width">
+    <h1>Delivering secure, high-quality <br> open source</h1>
+    <p>We empower anyone, anywhere, to access the best of open source <br> technologies</p>
+    <p><a href="" class="p-button--positive">Get in touch</a></p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>About Canonical</h2>
+  </div>
+  <div class="row">
+    <div class="col-4 u-equal-height">
+      <div class="p-card--highlighted has-accent-alt-border">
+        <h3 class="p-heading--4">Careers</h3>
+        <p class="p-card__content u-sv2">We are working on today's most important open source projects, with some of the greatest people around the globe with a truly distributed, diverse and passionate team.</p>
```

Comment:
Instead of using `u-sv2` It would be better to do the contents of the card like this:
```html
<div class="p-card__content">
  <p>We are working on today's most important open source projects, with some of the greatest people around the globe with a truly distributed, diverse and passionate team.</p>
  <p><a href="">Join us&nbsp;&rsaquo;</a></p>
</div>
```